com.unity.ml-agents/Runtime/Sensors/Reflection/IntReflectionSensor.cs

```diff
@@ -0,0 +1,15 @@
+namespace Unity.MLAgents.Sensors.Reflection
+{
+    internal class IntReflectionSensor : ReflectionSensorBase
+    {
+        internal IntReflectionSensor(ReflectionSensorInfo reflectionSensorInfo)
+            : base(reflectionSensorInfo, 1)
+        {}
+
+        internal override void WriteReflectedField(ObservationWriter writer)
+        {
+            var intVal = (System.Int32)GetReflectedValue();
+            writer[0] = intVal;
```

Comment:
Make the conversion to float explicit here ? 
```suggestion
            writer[0] = (float)intVal;
```

Reply:
Done